lib/killbill_client/models/combo_transaction.rb

```diff
@@ -23,6 +23,8 @@ def credit(user = nil, reason = nil, comment = nil, options = {}, refresh_option
       private
 
       def combo_payment(user, reason, comment, options, refresh_options = nil)
+        follow_location = true
+        follow_location = options.delete(:follow_location) if options.has_key?(:follow_location)
```

Comment:
Also should `:follow_location` be part of `options` or `refresh_options` ?
 
Oh i see, scratch that comment: `refresh_options`  is used for GET call.